gateway-manager/src/manage_home_app.py

```diff
@@ -64,7 +65,12 @@ def start_app():
 
     @app.route('/<realm>/', methods=['GET'])
     def index(realm):
-        return render_template('index.html', base_host=BASE_HOST)
+        services = []
+        url = f'{KONG_INTERNAL_URL}/services'
+        if not _check_404(url):
+            res = request(method='get', url=url)
+            services = res['data'] if 'data' in res else []
```

Comment:
you could return only the names and not the whole list... 

also you may filter the services by realm, not all of them could be enabled in this realm. Sadly for that point you need to check the service routes and take a look at the tags.
```python
            data = res.get('data', [])
            services = [
                service['name']
                for service in data
                if _check_service_in_realm(realm, service['name'])
            ]


def _check_service_in_realm(realm, name):

    def _realm_in_route(route):
        # json -> null will be returned as None, despite the default so we check.
        if route.get('tags', []) is not None:
            return realm in route.get('tags', [])
        else:
            return route['name'].endswith(f'__{realm}')

    try:
        # check all linked routes
        next_url = f'{KONG_INTERNAL_URL}/services/{name}/routes'
        while next_url:
            res = request(method='get', url=next_url)
            next_url = res['next']

            for route in res['data']:
                if _realm_in_route(route):
                    return True
    except HTTPError:
        pass

    return False
```

https://github.com/eHealthAfrica/gateway-manager/blob/master/gateway-manager/src/manage_kong.py#L234

As a final comment, to avoid the same error we already had in Gather, take in mind that `res = request(method='get', url=url)` is paginated and you need to check `res['next']`